File: Models/BusinessInProgress.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace MtpApp.Models
{
    public class BusinessInProgress
    {
        public int Id { get; set; }
        [Required]
        public int MonitoringId { get; set; }
        public Monitoring Monitoring { get; set; }
        [Required]
        public int BusinessSetUpCategoryId { get; set; }
        public BusinessSetUpCategory BusinessSetUpCategory { get; set; }
         [Required]
        public String StillInbusiness { get; set; }
         [Required]
         [StringLength(50)]
        public string BusinessType { get; set; }
         [Required]
         [StringLength(50)]
        public string Expense { get; set; }
         [Required]
         [StringLength(50)]
        public string Income { get; set; }
         [StringLength(255)]
        public string Note { get; set; }
    }
}

File: spring-backend/src/main/java/com/mtp/api/models/BusinessInProgress.java
package com.mtp.api.models;

import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Entity
@Table(name = "BusinessInProgresses")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class BusinessInProgress {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "MonitoringId")
    private Monitoring monitoring;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "BusinessSetUpCategoryId")
    private BusinessSetUpCategory businessSetUpCategory;

    @NotNull
    private String stillInbusiness;

    @NotNull
    @Size(max = 50)
    private String businessType;

    @NotNull
    @Size(max = 50)
    private String expense;

    @NotNull
    @Size(max = 50)
    private String income;

    @Size(max = 255)
    private String note;
}
